src/components/Cards/card-data/terra-station-cards.js

```diff
@@ -1,66 +1,59 @@
 const terraStationCards = [
   {
-    href: './download/',
+    href: '/learn/terra-station/download/',
     lightSvg: '/img/icons/station/station.svg',
     darkSvg: '/img/icons/dark/station.svg',
     header: 'Download Station',
     summary: 'Download Terra Station, the official wallet for Terra.'
```

Comment:
I think grammatically "the official wallet for Terra" 
should be "the official wallet of Terra."

Or maybe:
"Download Terra Station, Terra's official wallet."